<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>350</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>250</width>
    <height>350</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>250</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>231</width>
     <height>281</height>
    </rect>
   </property>
   <property name="title">
    <string>정보입력</string>
   </property>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>234</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>성      별 :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>전      공 :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>이      름 :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>전화번호 :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>80</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>비밀번호 :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>주      소 :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtname">
    <property name="geometry">
     <rect>
      <x>82</x>
      <y>50</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtpasswd">
    <property name="geometry">
     <rect>
      <x>82</x>
      <y>80</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtmajor">
    <property name="geometry">
     <rect>
      <x>82</x>
      <y>140</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtphone">
    <property name="geometry">
     <rect>
      <x>82</x>
      <y>170</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtaddress">
    <property name="geometry">
     <rect>
      <x>82</x>
      <y>200</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtgender">
    <property name="geometry">
     <rect>
      <x>82</x>
      <y>230</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtbirthday">
    <property name="geometry">
     <rect>
      <x>82</x>
      <y>110</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>생년월일 :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtId">
    <property name="geometry">
     <rect>
      <x>82</x>
      <y>20</y>
      <width>131</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>56</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>학      번 :</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnSignUp">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>310</y>
     <width>75</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>가입</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>btnSignUp</tabstop>
  <tabstop>txtname</tabstop>
  <tabstop>txtpasswd</tabstop>
  <tabstop>txtbirthday</tabstop>
  <tabstop>txtmajor</tabstop>
  <tabstop>txtphone</tabstop>
  <tabstop>txtaddress</tabstop>
  <tabstop>txtgender</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
